ci/rubyonrails.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This workflow will install a prebuilt Ruby version, install dependencies, and
+# run tests and linters.
+name: "Ruby on Rails CI"
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_DB: rails_test
+          POSTGRES_USER: rails
+          POSTGRES_PASSWORD: password
+    steps:
```

Comment:
Specify these values as [environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables) and use that across steps.

Reply:
Why should these be environment variables? The entire point is that they should not be configured. The database is configured with the exact values that are hardcoded in the test runner step.

If someone wants to configure their workflow to use different values, that is fine, they can do that! This is a template. But the point is that this action should work immediately, without making any changes to the user's system, and without making any changes to the Rails application code or configuration.